tests/Egret.Tests/Serialization/Audacity/AudacityImporterTests.cs

```diff
@@ -0,0 +1,193 @@
+        private readonly AudacitySerializer audacitySerializer;
+        private readonly ConfigDeserializer configDeserializer;
+
+        private const string ExampleProjectFile1 = @"Serialization\Audacity\example1.aup";
+
+        public AudacityImporterTests(ITestOutputHelper output) : base(output)
+        {
+            this.audacitySerializer = this.AudacitySerializer;
+            this.configDeserializer = BuildConfigDeserializer();
+        }
+
+        [Theory]
+        [FileData(ExampleProjectFile1)]
+        public async Task TestAudacitySerializer(string filePath)
+        {
+            var result = await audacitySerializer.Deserialize((FileInfoBase) new FileInfo(filePath));
+
+            result.Rate.Should().Be(44100.0);
+
+            result.Tags.Should().HaveCount(8);
+            result.Tags.Should().ContainInOrder(
+                new Tag("ARTIST", "artist"),
+                new Tag("TITLE", "track"),
+                new Tag("COMMENTS", "comments"),
+                new Tag("ALBUM", "album"),
+                new Tag("YEAR", "year"),
+                new Tag("TRACKNUMBER", "track number"),
+                new Tag("GENRE", "genre"),
+                new Tag("Custom", "Custom metadata tag")
+            );
+
+            result.Tracks.Should().HaveCount(2);
+
+            result.Tracks.First().Should().Match<LabelTrack>(i =>
+                i.Name == "Track 2" &&
+                i.IsSelected == 1 &&
+                i.Height == 206 &&
+                i.Minimized == 0 &&
+                i.Labels.Count == 2
+            );
+            result.Tracks.First().Labels.First().Should().Match<Label>(i =>
+                i.Title == "test 1" &&
+                i.TimeStart.AlmostEqual(4) &&
+                i.TimeEnd.AlmostEqual(7.2446258503) &&
+                i.SelLow.AlmostEqual(1) &&
+                i.SelHigh.AlmostEqual(10)
+            );
+            result.Tracks.First().Labels.Skip(1).First().Should().Match<Label>(i =>
+                i.Title == "test 3" &&
+                i.TimeStart.AlmostEqual(15.9714285714) &&
+                i.TimeEnd.AlmostEqual(24.44) &&
+                i.SelLow.AlmostEqual(10) &&
+                i.SelHigh.AlmostEqual(10000)
+            );
+
+            result.Tracks.Skip(1).First().Should().Match<LabelTrack>(i =>
+                i.Name == "Track 1" &&
+                i.IsSelected == 1 &&
+                i.Height == 90 &&
+                i.Minimized == 0 &&
+                i.Labels.Count == 1
+            );
+            result.Tracks.Skip(1).First().Labels.First().Should().Match<Label>(i =>
+                i.Title == "test 2" &&
+                i.TimeStart.AlmostEqual(8.0228571429) &&
+                i.TimeEnd.AlmostEqual(18.98) &&
+                i.SelLow == 0 &&
+                i.SelHigh == 0
+            );
+        }
+        
+        public static readonly TestFile HostConfig = ("/abc/host.egret.yaml", @"
+test_suites:
+  host_suite:
+    tests:
+      - file: bird*.wav
```

Comment:
this line of the config should be redundant

in an abstract sense: the main egret config is importing other tests (e.g. from another egret file, or an aup file acting like an egret file) and those tests should be referencing audio files.